models/marts/enriched_history_operations.yml

```diff
@@ -3,12 +3,24 @@ version: 2
 models:
   - name: enriched_history_operations
     description: Aggregate table for the history operations
+    tests:
+      - elementary.freshness_anomalies:
+          timestamp_column: closed_at
+          backfill_days: 7
+          time_bucket:
+            period: hour
+            count: 12
     columns:
       - name: op_id
         description: '{{ doc("operation_id") }}'
         tests:
-          - not_null
-          - unique
+          - unique:
+              config:
+                where: closed_at >= timestamp_trunc(timestamp_sub(current_timestamp(), interval 2 day), day) 
+                        and closed_at < timestamp_trunc(current_timestamp(), day)
+          - not_null:
+              config:
+                where: batch_run_date > current_datetime - interval 2 day
           - relationships:
               to: ref('stg_history_operations')
               field: op_id
```

Comment:
this is potentially a very expensive query to run if there are no date filters on the `stg_history_operations` table. Do you know if we have the ability to add a where clause that would limit the join?

Reply:
Although this test has been removed, I think we could use a where clause to limit the join. We just need to add "config: where:  " and the statement. More information on https://docs.getdbt.com/reference/resource-configs/where